doc/input-output-reference/src/overview/group-coil-system-cooling-water.tex

```diff
@@ -0,0 +1,171 @@
+
+\begin{itemize}
+\item HVAC, Average, Coil System Water Part Load Ratio {[]}
+\item HVAC, Average, Coil System Water Total Cooling Rate {[}W{]}
+\item HVAC, Average, Coil System Water Sensible Cooling Rate {[}W{]}
+\item HVAC, Average, Coil System Water Latent Cooling Rate {[}W{]}
+\item HVAC, Average, Coil System Water Control Status {[]}
+\end{itemize}
+
+
+\paragraph{Coil System Water Part Load Ratio {[]}}\label{coil-system-water-part-load-ratio}
+
+This output variable is the ratio of the sensible cooling load to the current full cooling capacity of the coil system. This variable reports the average load met as a fraction of the full coil capacity during the system timestep. If the ratio is 0.0, then there is no cooling load, else if the ratio is 1.0, then the load met is equal to the coil system full capacity.
+
+\paragraph{Cooling Coil Total Cooling Rate {[}W{]}}
+
+This output field is the total (sensible + latent) cooling rate of the coil system from the supply or outdoor air in Watts. This value is calculated using the enthalpy difference of the coil system outlet air and inlet air streams and the air mass flow rate through the coil system. This value is reported for each HVAC system timestep being simulated and is an average for the timestep.
+
+\paragraph{Cooling Coil Sensible Cooling Rate {[}W{]}}
+
+This output field reports the moist air sensible cooling rate of the coil system from the supply or outdoor air system. This value is calculated using the enthalpy difference of the coil system outlet air and inlet air streams at a constant humidity ratio, and the air mass flow rate through the coil system. This value is reported for each HVAC system timestep simulated and is an average for the timestep.
+
+\paragraph{Cooling Coil Latent Cooling Rate {[}W{]}}
+
+This output field is the latent cooling (dehumidification) rate of the coil system in Watts. This value is calculated as the difference between the total cooling rate and the sensible cooling rate provided by the coil system. This value is reported for each HVAC system timestep being simulated and is an averaged for the timestep.
+
+\paragraph{Coil System Water Control Status {[]}}\label{coil-system-water-control-status}
+
+This output field indicates whether the coil system is favorable to operate or not. Control status value of 1 indicates that the condition is favorable for the coil system to operate. Control status value of 0 indicates the condition is not favorable the coil system to operate. The control status is determined from the coil entering air temperature, coil entering water temperatures and user specified temperature offset. If the coil entering air temperature is above the coil entering water temperatures by more than the specified temperature offset, then the control status is set to 1, else it is set to 0. This value is reported for each HVAC system timestep being simulated, and the control status is an average for the timestep.
+
```

Comment:
Input output reference document for **CoilSystem:Cooling:Water** object with diagrams.